src/DurableTask.Netherite/OrchestrationService/NetheriteOrchestrationServiceSettings.cs

```diff
@@ -47,7 +47,7 @@ public class NetheriteOrchestrationServiceSettings
 
         /// <summary>
         /// Optionally, a name for an Azure Table to use for publishing load information. If set to null or empty,
-        /// then Azure blobs are used instead. The use of Azure blobs is currently not supported on consumption plans, or on elastic premium plans without runtime scaling.
+        /// then Azure blobs are used instead.
```

Comment:
is this still accurate?

Reply:
Good question. I don't quite remember, this was a limitation of the scale controller (couldn't read anything other than Azure Tables). Not sure if we changed that since then.